docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1975 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #58.            is_cands, _ = _update_is_cands(T[i:i+m], T, M_T[i], Σ_T[i], M_T, Σ_T, min_dist, 
</pre><p>Can we please revert this back?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
Yes.